content/en/blog/_posts/XXXX-XX-XX-volume-attributes-class.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: blog
+title: "Kubernetes 1.31: VolumeAttributesClass for Volume Modification Beta"
+date: XXXX-XX-XX
```

Comment:
@mattcary please update the publication date to August 15. thanks

Reply:
done